rundeckapp/grails-app/services/rundeck/services/ApiService.groovy

```diff
@@ -915,12 +911,27 @@ class ApiService implements WebUtilService{
     @Transactional
     def removeAllExpiredTokens() {
         def now = Date.from(Clock.systemUTC().instant())
-        def found = AuthToken.findAllByExpirationLessThan(now)
+        def found = tokenProvider.findAllByExpirationLessThan(now)
```

Comment:
I wonder if this method should be moved to the TokenDataProvider interface, so that the provider could do it more efficiently

Reply:
can wait until next phase